README.md

```diff
@@ -1,4 +1,30 @@
 
-# **vine-service**
+# Vine Service
 
-Core microservice for PET-project
\ No newline at end of file
+Spring boot application that provides an API for making operation with vine project:
+- create vine
+- get vines
+- create order
+
+## Project structure
+
+- [vine controller](src/main/java/com/mentorship/vineservice/controller/VineController.java) 
+- [domain objects ](src/main/java/com/mentorship/vineservice/domain)
+- [service implementation layer](src/main/java/com/mentorship/vineservice/service/impl) 
+- [repository layer](src/main/java/com/mentorship/vineservice/repository)
+
+## Service Requirements
+
+Requires:
+- a running [Eureka registry](https://github.com/Volodymyr2907/vine-registration-service);
+- a running RabbitMQ broker;
+- a running Mysql DB;
+- a running [user service](https://github.com/Volodymyr2907/user-service) (for interacting with permission)
+- a running [messaging service](https://github.com/Volodymyr2907/notification-service) (for handling events)
+
+## How to run
```

Comment:
Let's add an information about configuration, which properties and where should be set/changed